bin/generators/plugins/configure.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
+const parentRequire = require('parent-require');
+const eg = require('../../eg');
+const PluginInstaller = require('../../../lib/plugin-installer');
+
+module.exports = class extends eg.Generator {
+  constructor (args, opts) {
+    super(args, opts);
+
+    this.installer = null;
+    this.pluginOptions = null;
+
+    this.configureCommand({
+      command: 'configure <package>',
+      description: 'Configure a plugin',
+      builder: yargs =>
+        yargs
+          .usage(`Usage: $0 ${process.argv[2]} configure <package>`)
+          .example(`$0 ${process.argv[2]} configure url-rewrite`)
+    });
+  }
+
+  initializing () {
+    this.packageName = this.argv.package;
+
+    if (!this.packageName.startsWith(PluginInstaller.PACKAGE_PREFIX)) {
+      try {
+        const prefixedName = PluginInstaller.PACKAGE_PREFIX + this.packageName;
+        this.pluginManifest = this._getPluginManifest(prefixedName);
+        this.packageName = prefixedName;
+      } catch (_) {
+        // This will fall through to the next conditional statement.
+      }
+    }
+
+    if (!this.pluginManifest) {
+      try {
+        this.pluginManifest = this._getPluginManifest(this.packageName);
+      } catch (_) {
+        this.log.error('Plugin not installed:', this.packageName);
+        process.exit();
+      }
+    }
+
+    this.installer = PluginInstaller.create({
+      packageName: this.packageName,
+      pluginManifest: this.pluginManifest
+    });
+  }
+
+  prompting () {
+    const optionsMeta = this.installer.pluginManifest.options || {};
+    const keys = Object.keys(optionsMeta);
+
+    const previousPluginOptions = this.installer.existingPluginOptions;
+
+    const pluginQuestions = keys.map(key => {
+      const schema = optionsMeta[key];
+      return {
+        type: 'input',
+        name: `pluginOption${key}`,
+        message: `Set value for ${key}:`,
+        default: previousPluginOptions[key],
+        validate: input => {
+          const type = schema.type;
+
+          if (['string', 'boolean', 'number'].indexOf(type) === -1) {
+            this.log.error(
+              `Invalid plugin option: ${key}. Type must be string, boolean, ` +
+              'or number.');
+
+            return false;
+          }
+
+          if (schema.required && !input) {
+            return false;
+          }
+
+          if (type === 'number' && isNaN(input)) {
```

Comment:
integer is also valid JSON schema type as well as array (ip addresses etc.). 
I'm not even speaking for options like "minimum" and validation of array items 

So it may be ok for first version, but I would prefer using some validator like  https://www.npmjs.com/package/ajv or any other

Reply:
Yes, we can increase JSON Schema support on a future iteration.